### DISCUSSION TITLE
breacking point issue

### DISCUSSION THREAD

**Discussion by Mostafa-Mohammadi** [tailwindlabs/tailwindcss#2702] [Help]
i have writing some custome css style for my svg navbar , but when i try add sm mode then my lg not working  like this

 class="w-full sm:navbar-sm lg:navbar-lg pb-24 "

**Comment by ghost** [comment]
repo? - this might help - https://github.com/tailwindlabs/tailwindcss/discussions/2693#discussioncomment-114588

**Comment by simonswiss** [comment]
Hey @kyle-hensoon 👋

Can you please try to reproduce your scenario in a new https://play.tailwindcss.com example? When you're done, you can click the `share` button and copy the link in here.

My wild guess is that you're not wrapping your custom CSS in a [`@responsive` directive](https://tailwindcss.com/docs/functions-and-directives#responsive), and therefore the `md`, `lg` etc variants are not being generated.

Would love to see your what the code in. your CSS file looks like 👍
